### PR TITLE
feat: playground updates

### PR DESCRIPTION
**Pull request by fakepixels**
**What changed? Why?**

Screenshot 2024-10-11 at 5 45 32 PM

- Major UI update to Playground

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +34,13 @@
     "react-dom": "^18"
   },
   "dependencies": {
+    "@coinbase/onchainkit": "^0.33.5",
```
Comment (cpcramer) [inline package.json:37]: We should remove these changes in this package.json file!

File: playground/nextjs-app-router/app/globals.css [hunk 2]
```diff
@@ -2,6 +2,8 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap');
```
Comment (cpcramer) [inline playground/nextjs-app-router/app/globals.css:5]: We can create an `index -with-tailwind.css` file for all css imports. Similar to what we do in src

File: playground/nextjs-app-router/app/globals.css [hunk 3]
```diff
@@ -57,13 +59,14 @@
...
 
-@layer base {
   * {
     @apply border-border;
   }
+
   body {
-    @apply bg-background text-foreground;
+    @apply text-foreground;
+    background-color: hsl(var(--background)); /* Define bg-background */
```
Comment (cpcramer) [inline playground/nextjs-app-router/app/globals.css:69]: Once theming is merged we will be able to update this var to the `--ock` one

File: playground/nextjs-app-router/components/Demo.tsx [hunk 4]
```diff
@@ -82,58 +79,82 @@ function Demo() {
...
+        <button
+          type="button"
+          onClick={toggleDarkMode}
+          className="rounded-full border p-2 text-xl shadow-lg transition-colors hover:bg-gray-100 dark:bg-gray-800 dark:hover:bg-gray-700"
+        >
+          {isDarkMode ? (
+            <Sun className="text-yellow-500" />
+          ) : (
+            <Moon className="text-blue-500" />
+          )}
```
Comment (cpcramer) [inline playground/nextjs-app-router/components/Demo.tsx:92]: Nice!

**Comment by fakepixels** [comment]
This version isn't mobile friendly yet. The ideal version is to have the floating sidebar island be a drawer on mobile with the option to close the drawer to give space to view the component. cc @alissacrane-cb @alessey for help.

**Comment by alessey** [comment]
currently this doesn't work on mobile 
image

**Pull request closed without merging**